Documentation/compatibility/winforms-accessibility-changes-472.md

```diff
@@ -0,0 +1,107 @@
+## Accessibility improvements in Windows Forms controls for .NET 4.7.2
+
+### Scope
+Major
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Windows Forms Framework is improving how it works with accessibility technologies to better support Windows Forms customers. These include the following changes:
+- Changes to improve display during High Contrast mode.
+- Changes to improve the keyboard navigation in the DataGridView and MenuStrip controls. 
+- Changes to interaction with Narrator.
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+__How to opt in or out of these changes__
+  
+In order for the application to benefit from these changes, it must run on the .NET Framework 4.7.2 or later. The application can benefit from these changes in either of the following ways:
+- It is recompiled to target the .NET Framework 4.7.2. These accessibility changes are enabled by default on Windows Forms applications that target the .NET Framework 4.7.2 or later.
+- It opts out of the legacy accessibility behaviors by adding the following [AppContext Switch](https://docs.microsoft.com/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) to the ```<runtime>``` section of the app config file and setting it to false, as the following example shows.
+```
+    <?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+      <startup>
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7"/>
+      </startup>
+      <runtime>
+        <!-- AppContextSwitchOverrides value attribute is in the form of 'key1=true|false;key2=true|false  -->
+        <AppContextSwitchOverrides value="Switch.UseLegacyAccessibilityFeatures=false;Switch.UseLegacyAccessibilityFeatures.2=false" />
+      </runtime>
+    </configuration>
+```
+Applications that target the .NET Framework 4.7.2 or later and want to preserve the legacy accessibility behavior can opt in to the use of legacy accessibility features by explicitly setting this AppContext switch to ```true```.
+
+__Use of OS-defined colors in High Contrast themes__
+- The drop down arrow arrow of the `T:System.Windows.Forms.ToolStripDropDownButton` now has uses now use OS-defined colors in High Contrast theme.

```

Comment:
These uuids don't resolve when migrated. They have to be in xref format:  \<xref:System.Windows.Forms.ToolStripDropDownButton>. That's true of all of the other `T:...` references here.
now has uses now use --> now uses

Reply:
Is this xref format different from last release?  Do I remove all the T: and P: and just use the xref?

Reply:
And do I keep the fence ` around the xref?